connectors/src/connectors/slack/index.ts

```diff
@@ -580,13 +590,13 @@ export class SlackConnectorManager extends BaseConnectorManager<SlackConfigurati
     const channels = await SlackChannel.findAll({
       where: {
         connectorId: this.connectorId,
-        slackChannelId: internalIds,
+        slackChannelId: slackChannelIds,
       },
     });
 
     const contentNodes: ContentNode[] = channels.map((ch) => ({
       provider: "slack",
-      internalId: ch.slackChannelId,
+      internalId: internalIdFromSlackChannelId(ch.slackChannelId),
```

Comment:
is it the right time to add a Resource with a method `toContentNode`?

Reply:
actually if this moves to core maybe not

Reply:
Yep this is all going away :-)